fetch.js

```diff
@@ -341,7 +370,19 @@
   Body.call(Response.prototype)
 
   Response.prototype.clone = function() {
-    return new Response(this._bodyInit, {
+    var clone
+
+    if (this.bodyUsed) {
+      throw new TypeError('Failed to execute \'clone\' on \'Response\': Response body is already used')
```

Comment:
Could as well fall thought and use tee()'s error message that already checks if stream is locked